data/templates/run_end_to_end_tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Script to run end-to-end tests on Ubuntu Jenkins instance.
+#
+# This file is generated by l2tdevtools update-dependencies.py.
+
+# Fail on error.
+set -e
+
+CONFIGURATION=$$1;
+
+CONFIGURATION_FILE="$${CONFIGURATION}.ini";
```

Comment:
Why have this variable and not use it?

Reply:
It is used, if on line 31 this variable is checked. If the file exists it used,